python-frontend/typeshed_serializer/tests/conftest.py

```diff
@@ -47,6 +47,11 @@ def typeshed_custom_stubs():
     assert len(build_result.errors) == 0
     return build_result
 
+@pytest.fixture(scope="session")
+def microsoft_stubs():
+    build_result, _ = MicrosoftStubsSerializer().get_build_result()
+    assert len(build_result.errors) == 323
```

Comment:
Nice, should we have a ticket or something in place to solve that ?

Reply:
I don't think we need to fix this as they are all import  error as mypy tries to solve everything, even non relevant imports.